src/test/scala/edu/ie3/simona/model/participant/ChpModelSpec.scala

```diff
@@ -236,29 +238,32 @@ class ChpModelSpec
     "Check storage level after calculating next state with #chpState and heat demand #heatDemand kWh:" in {
       val testCases = Table(
         ("chpState", "storageLvl", "heatDemand", "expectedStoredEnergy"),
-        (chpStateNotRunning, 90, 0, 1035), // tests case (false, false, true)
+        (chpStateNotRunning, 70, 0, 805), // tests case (false, false, true)
         (
           chpStateNotRunning,
-          90,
+          70,
+          // todo check this
           8 * 115,
-          230,
+          0,
         ), // tests case (false, true, false)
-        (chpStateNotRunning, 90, 10, 1025), // tests case (false, true, true)
-        (chpStateRunning, 90, 0, 1135), // tests case (true, false, true)
-        (chpStateRunning, 90, 8 * 115, 230), // tests case (true, true, false)
-        (chpStateRunning, 90, 10, 1125), // tests case (true, true, true)
-        (
-          chpStateRunning,
-          90,
+        (chpStateNotRunning, 70, 10, 795), // tests case (false, true, true)
+        // todo check this, why 1135?
+        (chpStateRunning, 70, 0, 905), // tests case (true, false, true)
+        (chpStateRunning, 70, 8 * 115, 0), // tests case (true, true, false)
+        (chpStateRunning, 70, 10, 895), // tests case (true, true, true)
+        // todo, why not +100?
+        (chpStateRunning,
+          70,
           806,
-          329,
+          99,
         ), // test case (_, true, false) and demand covered together with chp
         (
           chpStateRunning,
-          90,
+          70,
           9 * 115,
-          230,
+          0,
         ), // test case (_, true, false) and demand not covered together with chp
+        // todo check this, why 92?
```

Comment:
imho: If the storage contains 92 m³ this represents 1058 kWh. + 100 kWh from Chp => 1158 which is more than 1150 of the storage volume. Thus explains in my opinion the comment below, (chp) test case AND  storage volume exceeds maximum. Also I'm still not fully happy with the comment since it's hard to understand. If you agree or have a better formulation, please go ahead.

Reply:
Maybe: "// test case (true, true, true) CHP running, storage at 92% (1058 kWh) + 100 kWh from CHP exceeds max capacity (1150 kWh)." ?